arcgis-ios-sdk-samples/Layers/Add ENC exchange set/AddENCExchangeSetViewController.swift

```diff
@@ -0,0 +1,108 @@
+// Copyright 2020 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class AddENCExchangeSetViewController: UIViewController {
+    /// The map view managed by the view controller.
+    @IBOutlet weak var mapView: AGSMapView! {
+        didSet {
+            mapView.map = AGSMap(basemap: .oceans())
+        }
+    }
+    
+    /// Load the ENC dataset and add to the map view.
+    ///
+    /// - Parameter mapView: The map view managed by the view controller.
+    func addENCExchangeSet(mapView: AGSMapView) {
+        guard let map = mapView.map else { return }
+        // Load catalog file in ENC exchange set from bundle.
+        let catalogURL = Bundle.main.url(
+            forResource: "CATALOG",
+            withExtension: "031",
+            subdirectory: "ExchangeSetwithoutUpdates/ExchangeSetwithoutUpdates/ENC_ROOT"
```

Comment:
Any chance we could get the portal item updated so that there aren't two `ExchangeSetwithoutUpdates` directories? (Bonus points for getting the _w_ capitalized!) That just looks really odd.